docs/DeveloperGuide.md

```diff
@@ -236,42 +236,124 @@ _{Explain here how the data archiving feature will be implemented}_
+| `* *` | Forgetful Tutor | Track all my tasks | Know which is of greater urgency |
+| `* *` | Busy Tutor | Store zoom links | Retrieve these zoom links for my lessons, consultations, etc. |
+| `* *` | Busy Tutor | Store class rooms | Be reminded of my lesson venues |
+| `* *` | Tutor with many students | Keep notes on each student&#39;s performance | Track their progress over time |
+| `* *` | Careless tutor | Undo my commands | Correct any errors when I input things wrongly |
+| `* *` | Tutor | Set recurring events (eg. lessons for every week) | Avoid typing the same events |
+| `* *` | Tutor | Calculate the mean, median, mode of my students' scores | Gauge the overall performance of my class |
+| `* *` | Tutor | Store the questions asked by students | Provide students with answers immediately, for questions that were asked before |
+| `* *` | Tutor | Be reminded of my lessons | Attend them |
+| `* *` | Tutor | Update the information of my students | Update my understanding of the progress of my students |
+| `* *` | Tutor | Categorise my students into how well they are doing | Dedicate more time towards the weaker students |
+| `* *` | Careless Tutor | Redo my undone actions | Easily reverse my accidental undos. |
+| `*` | Caring tutor | Take note of student&#39;s special needs, if any | Cater my teaching toward them |
+| `*` | Tutor for many semesters | Archive my past semesters | Avoid cluttering the app |
+| `*` | Tutor | Store teaching feedback given by my tutees | Improve my teaching |
+| `*` | Tutor | Prioritise my tasks | Work on important tasks first |
+| `*` | Tutor | Keep track of the hours I have spent teaching/preparing for class | Be aware of how much time I have spent on teaching |
+| `*` | Tutor | Store teaching feedback given by Professors | Improve my teaching |
+| `*` | Forgetful Tutor | Set an alert for task deadlines | Complete my tasks on time |
+| `*` | Forgetful Tutor | Store picture of my students | Easily match their faces to their names |
+| `*` | Tutor with many classes | See all tasks related to a class | Easily tell which task belongs to which class |
+| `*` | Tutor teaching mods that require many written assignments | View my student&#39;s written submissions | Mark/review their homework |
+| `*` | Tutor | Manage my teaching materials | Find them easily |
+| `*` | Tutor | Find free time | Provide consultation for students |
 
 ### Use cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
+(For all use cases below, the **System** is `Tutor's Pet` and the **Actor** is the `user`, unless specified otherwise)
+
+**Use case: UC01 - Edit a student**
+
+**MSS**
+
+1.  User requests to list students
+2.  Tutor's Pet shows a list of students
+3.  User requests to edit a specific student in the list
+4.  User provides the parameters that are to be edited
+5.  Tutor's Pet edits the student's contact information 
```

Comment:
Perhaps it will be better to remove `contact` since users can edit tags as well. And tags may not be contact information.